mir_eval/separation.py

```diff
@@ -328,7 +434,8 @@ def _safe_db(num, den):
     return 10 * np.log10(num / den)
 
 
-def evaluate(reference_sources, estimated_sources, **kwargs):
+def evaluate(reference_sources, estimated_sources,
+             window=None, hop=None, **kwargs):
```

Comment:
Assuming `window` and `hop` indeed can/should have default values, `window` and `hop` should not be kwargs, they should be passed in **kwargs and you should use `util.filter_kwargs` (see other `evaluate` functions for examples of this).  Then you can avoid all of the special-casing, and just iterate over all of the functions in the submodule.  The way it is is a little incon